src/getList/index.ts

```diff
@@ -12,44 +11,53 @@ export type GetList<R> = (conf: {
 
 export type Search<R> = (
   q: string,
-  limit: number,
-  filter: Record<string, any>
+  conf: {
+    limit: number
+    offset: number
```

Comment:
I agree that this signature is great because having named attributes avoid mixing them (especially when they have the same type) but this generate a breaking change which is also to be considered: it means all the users will have to check changelog and possibly adjust their code. Do es it worth it?